package.json

```diff
@@ -21,7 +21,7 @@
     "build:package-json": "node scripts/package.cjs",
     "build:svelte": "svelte-package",
     "build:static": "cp .npmignore README.md LICENSE dist/",
-    "build:tsc": "tsc tailwind.config.ts --outDir dist --declaration",
+    "build:tsc": "tsc tailwind.config.ts src/jsx.d.ts --outDir dist --declaration",
```

Comment:
The version update of related react stuff broke the types here. So added this back into the types